guides/k8s-microservices/src/main/paradox/index.md

```diff
@@ -0,0 +1,369 @@
+| [Service](https://kubernetes.io/docs/concepts/services-networking/service/)                             | Provides the means to expose TCP and UDP ports to other Pods within the Kubernetes cluster, and it integrates with DNS so they can be discovered via DNS SRV.                                                                                                                                                                                                                     |
+| [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)                             | A collection of rules that allow external traffic to reach services running inside Kubernetes. This enables, for example, requests to /api/users to be routed to the friendservice while requests for / are routed to web. It also provides a central place to terminate TLS. In this example, Chirper is configured to use nginx as the ingress controller and to terminate TLS. |
+
+_Refer to Chirper's resources at `deploy/kubernetes/resources` for more details._
+
+#### Service Location
+
+Lagom makes use of a [Service Locator](https://www.lagomframework.com/documentation/1.3.x/java/ServiceLocator.html) to
+call other services within the system. Chirper is configured to use the [service-locator-dns](https://github.com/typesafehub/service-locator-dns)
+project to provide a service locator that takes advantage of Kubernetes [Service Discovery](https://kubernetes.io/docs/concepts/services-networking/service/#discovering-services).
+
+Because the names of your service locators will not exactly match the DNS SRV address, `service-locator-dns` has
+the ability to translate addresses. Chirper uses this feature to ensure, for example, a service lookup for `friendservice`
+will be translated into a DNS SRV lookup for `_http-lagom-api._tcp.friendservice.default.svc.cluster.local`. Chirper
+is configured with the following in each of its service's `application.conf`:
+
+```
+service-locator-dns {
+  name-translators = [
+    {
+      "^_.+$" = "$0",
+      "^.*$" = "_http-lagom-api._tcp.$0.default.svc.cluster.local"
+    }
+  ]
+
+  srv-translators = [
+    {
+      "^_http-lagom-api[.]_tcp[.](.+)$" = "_http-lagom-api._http.$1",
+      "^.*$" = "$0"
+    }
+  ]
+}
+```
+
+_Refer to the various `application.conf` files in the Chirper repository for more details._
+
+### Deployment
+
+Now that all the resources required for deployment have been described, this guide will cover how to automate the process
+of deploying them to Kubernetes.
+
+Deploying Chirper requires the following actions:
+
+* Setup Kubernetes
+* Deploy Cassandra
+* Build Chirper Docker images
+* Deploy Chirper
+* Deploy nginx
+* Open the service in your browser
+
+Let's take a look at how these tasks can be performed from your own terminal. Make sure 
+you've `cd`'d into your clone of the Chirper repository before proceeding.
+
+##### Setting up your Kubernetes Cluster
```

Comment:
ktoso and fsat added numbering to their headings, which I think makes it easier to follow, they have divergent paths so used n.n format, but I think for this guide, maybe:

#### Step 1: Setting up your Kubernetes Cluster and etc, to match the list of steps in the overview.